client/src/main/java/org/zalando/nakadi/client/java/ClientImpl.java

```diff
@@ -51,47 +53,59 @@ public ClientImpl(JavaClientHandler handler) {
 
     @Override
     public Future<Void> createEventType(EventType eventType) {
+    	checkNotNull(eventType, "EventType may not be null");
```

Comment:
:-).
just one note: Java 8 provides Objects.requireNonNull() which is the same as the one provided by Guava